lib/style.css

```diff
@@ -101,6 +115,8 @@ body {
 }
 
 .material .song-row .song-indicator {
+  background-image: url('ani_equalizer_white_x2.gif');
+  background-size: 28px 28px;
```

Comment:
For some reason the white image is a slightly different scale to the black one, so I needed to make it smaller for it too visually match